scout/server/blueprints/institutes/templates/overview/cases.html

```diff
@@ -86,9 +86,10 @@
           <th style="width: 15%;">Analysis date <a href="" id="{{group_name}}_analysis_date" onclick="updateArgs('{{group_name}}','sort=analysis_date','{{sort_option}}');" class="badge" style="color:{{'black' if sort_by=='analysis_date' else 'grey'}};background-color:white;">
             <i class="fa fa-sort{{'-'+sort_order if sort_by=='analysis_date' }}" aria-hidden="true" data-bs-toggle="tooltip" title="Sort by analysis date"></i>
           </a></th>
-          <th style="width: 20%;">Link</th>
```

Comment:
You could also cut some space from the last column (sharing). Looks quite empty atm:

<img width="1270" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/13f8da30-9d00-4cf5-9536-7ed1c5932b1b">

Reply:
Both the Date and Sequencing column tend to have some space, but esp for the latter we may want the case ref genome displayed if different from the instance default..